lms/templates/static_templates/embedded_new_tab.html

```diff
@@ -11,15 +11,38 @@
             dataType: 'json',
             contentType: 'application/json; charset=utf-8',
             success: function(data) {
+                var isTimeExam = $('#time-exam').val();
                 if (data.cookie_sent) {
-                    window.location.href = window.location.href + $('#embedded-link').val();
+                    if (isTimeExam === '1') {
+                        var isInIframe = inIframe();
+                        if (isInIframe){
+                            showNewTabMsg();
```

Comment:
In this case message should differ from the message in case of browsers that block cookies